extensions/interactions/NumberWithUnits/directives/number_with_units_help_modal_directive.html

```diff
@@ -0,0 +1,149 @@
+<div class="modal-header">
+  <b>Guidelines for writing units</b>
+</div>
+<div class="modal-body">
+  <style>
+    td {
+      padding: 5px;
+      width: 30%;
+    }
+  </style>
+  <table class="table-bordered">
+    <tr>
+      <td><b>Rule Description</b></td>
+      <td><b>Incorrect Format</b></td>
+      <td><b>Correct Format</b></td>
+    </tr>
+    <tr>
+      <td>
+        Separate value and the unit with one space.
+      </td>
+      <td>3km</td>
+      <td>3 km</td>
+    </tr>
+    <tr>
+      <td>
+        In case the value is a fraction, separate numerator and denominator by
+        forward slash '/' symbol.
+      </td>
+      <td>3 by 5 km</td>
+      <td>3/5 km</td>
+    </tr>
+    <tr>
+      <td>
+        For multiplying two or more units, put one space or asterisk '*' between them.
```

Comment:
For the asterisk part, it looks like the same you have has a space, asterisk, and space, perhaps you might want to change it so that it says `...or the sequence of space, asterisk '*', and space between the units.`

Reply:
I've shown that way for clarity. Actually only asterisk between them is also allowed like- `kg*s` is also valid.

Reply:
Done.